packages/stack-shared/src/helpers/vault/client-side.ts

```
@@ -0,0 +1,82 @@
+  const valueEncryptionDerivedKey = await getDerivedKey(secret, key);
+
+  const bytes = await decrypt({
+    purpose: encryptionValuePurpose,
+    secret: valueEncryptionDerivedKey,
+    cipher: decodeBase64(encryptedValue),
+  });
+  return new TextDecoder().decode(bytes);
+}
+
+
+import.meta.vitest?.describe("encryptValue & decryptValue", ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)